### ISSUE TITLE
[Modular CLI]

### ISSUE THREAD

**Issue by phueac** [modularml/mojo#2265]
### Issue description

I am trying to install and use Mojo within a Docker container. Your blog post at https://www.modular.com/blog/how-to-setup-a-mojo-development-environment-with-docker-containers indicates this can be does by passing an AUTH_KEY into the container during build. But it looks like this authentication method was deprecated in a recent update to the Modular CLI? 

It seems from your example at https://github.com/modularml/mojo/tree/main/examples/docker that an auth key is still expected to build a Docker image. I found by trial and error that supplying a made-up string of random characters as the key allowed the Docker build to succeed.

Perhaps you need to update the blog post and Docker example to reflect the deprecation of the AUTH_KEY method?

As a side note, it is kind of sketchy to require a login to your proprietary system before even being able to download and try out your tools. How are you going to convince Python developers who are used to a completely open ecosystem to adopt this?

### Steps to reproduce

- Include relevant code snippet that did not work as expected.
- If applicable, add screenshots to help explain the problem.
- Include anything else that might help us debug the issue.

### Version Info

```shell
- Provide Modular CLI version by pasting the output of `modular -v`
- What OS did you install modular CLI on ?
```

**Comment by smuniraj2018** [comment]
Thanks, @phueac for your feedback. The recent client side changes in the auth flow change is a bug and we are working to fix this by the next release. I will forward you feedback on the remaining items to our internal teams.

**Comment by byteshiva** [comment]
I blogged about using Docker to run Mojo with auto-generated authentication keys. It's a quick fix for those who need to bypass authentication.

link: 
https://byteshiva.medium.com/elevating-your-workflow-dockerizing-mojo-for-seamless-development-628a0bc4b25b

**Comment by josiahls** [comment]
@phueac mojo nightly is accessible without an auth key via pixi: https://docs.modular.com/pixi/ . 
Example:
```
[dependencies]
mojo = ">=0.25.7.0.dev2025101518,<0.26.0.0"
```
Example repo: https://github.com/bgreni/EmberJson/blob/main/pixi.toml

I just need to add 3 run lines in my images: https://github.com/josiahls/c_binder_mojo/blob/master/c_binder_mojo.Dockerfile#L90-L94 

@JoeLoser I think this is solved.

**Comment by conorbronsdon** [comment]
Mojo installation has been completely reworked. You can now install via `pixi add mojo`, `uv pip install mojo` (with the [Modular index](https://docs.modular.com/mojo/manual/install)), or conda. The old `modular` CLI and AUTH_KEY are no longer needed. Thanks!

The blog post is likely out of date - hard to keep our older blogs up to date with changes in parallel with the code.
